### ISSUE TITLE
Timeline not updating UI after action

### ISSUE THREAD

**Issue by taylorpoe** [HaikuTeam/animator#8]
Summary:
Today I've started seeing some regressions in timeline ux. It appears that some actions aren't updating until the user vertically scrolls the timeline area. This has included adding keyframes, adding trans segments, moving keyframes. Here's a [video example](https://cl.ly/0j2D3n3J3K1k).

Steps to reproduce:
Move the middle keyframe of two conjoined transition segments 

Screenshots, logs, etc:
[moving keyframe](https://cl.ly/0j2D3n3J3K1k).

Release version, system info, etc:
2.0.105 staging; 2.0.104 prod

Account, project info, etc:
**ORG:** taylor
**PROJ:** mountainsillo

**Comment by taylorpoe** [comment]
After a bit more review, it appears that the issue only occurs when Creator has been running 5+ minutes. I made sure all of the should updates like [here](https://github.com/HaikuTeam/timeline/blob/master/src/components/Timeline.js#L1353) returned false. I did not encounter the Issue #8 after doing this, but can't say for sure it is the cause due to the factor of needing Creator to be open for 5+ minutes to recreate it in the first place.
